docs/user/integration/opentelemetry-demo/README.md

```diff
@@ -0,0 +1,197 @@
+# Install OpenTelemetry Demo Application in Kyma
```

Comment:
```suggestion
# Integrate OpenTelemetry Demo App with telemetry module
```
What about this?

Reply:
Ok then I suggest "Integrate OpenTelemetry Demo App" :D

Reply:
https://github.com/kyma-project/telemetry-manager/pull/586#:~:text=OTel%20Demo%20guide-,c4ed054,-Merge%20state